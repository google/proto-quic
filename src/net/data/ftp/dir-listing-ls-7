-rw-r--r--    1 0        100          3108 Mar 07  2001 00readme.html
drwxr-xr-x    3 1164     100          4096 Oct 19 13:45 OCU
lrwxrwxrwx    1 203      1              10 Jun 15  2006 año2000 -> ./urte2000
drwxr-xr-x    2 0        0            4096 Mar 07  2001 bin
drwxr-xr-x    2 0        100          4096 Mar 07  2001 dev
drwxr-xr-x    3 0        100          4096 Apr 20  2005 doc
