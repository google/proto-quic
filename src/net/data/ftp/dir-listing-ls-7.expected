-
00readme.html
3108
2001
3
7
0
0

d
OCU
-1
1994
10
19
13
45

l
a√±o2000
-1
2006
6
15
0
0

d
bin
-1
2001
3
7
0
0

d
dev
-1
2001
3
7
0
0

d
doc
-1
2005
4
20
0
0
